src/python/bot/tasks/progression_task.py

```diff
@@ -346,6 +346,18 @@ def find_fixed_range(testcase_id, job_type):
     if max_index - min_index == 1:
       _save_fixed_range(testcase_id, min_revision, max_revision)
       return
+    # TODO(mbarbella): Occasionally, we get into this bad state. It seems to be
```

Comment:
I dont think we need todo here, can just remove TODO and also "# Remove this if the root cause can be resolved elsewhere."